spec/requests/care_plan_spec.rb

```diff
@@ -0,0 +1,46 @@
+RSpec.describe "Care Plan", type: :request do
+  describe '#index' do
+    context 'when user is signed in' do
+      let(:user) { create(:user) }
+      before do
+        sign_in user
+        get '/care_plan'
+      end
+
+      it 'renders template' do
+        expect(response).to render_template(:index)
+      end
+
+      it 'does not render a different template' do
+        expect(response).to_not render_template(:show)
+      end
+
+      it 'response is successful' do
+        expect(response).to have_http_status(:ok)
+        expect(response.content_type).to eq('text/html; charset=utf-8')
+        expect(response).to be_successful
+      end
+
+      context 'when there are bookmared strategies' do
+        let(:strategy) { create(:strategy, user: user, bookmarked: true) }
+
+        it 'assigns @bookmarked_strategies' do
+          get '/care_plan'
+          expect(assigns(:bookmarked_strategies)).to eq([strategy])
```

Comment:
As pointed out by @bennpham in other PRs, `assigns` is part of a gem being removed from the project and should not be used. Request tests should be testing the output, so please instead check the `response.body` here. See other tests for examples

Reply:
it's solved